conf.d/python.d/httpcheck.conf

```diff
@@ -0,0 +1,83 @@
+
+# Autodetection and retries do not work for this plugin
+
+# ----------------------------------------------------------------------
+# JOBS (data collection sources)
+#
+# The default JOBS share the same *name*. JOBS with the same name
+# are mutually exclusive. Only one of them will be allowed running at
+# any time. This allows autodetection to try several alternatives and
+# pick the one that works.
+#
+# Any number of jobs is supported.
+#
+# -------------------------------
+# ATTENTION: Any valid configuration will be accepted, even if initial connection fails!
+# -------------------------------
+#
+# There is intentionally no default config, e.g. for 'localhost'
+
+# job_name:
+#     name: myname            # [optional] the JOB's name as it will appear at the
+#                             #   dashboard (by default is the job_name)
+#                             #   JOBs sharing a name are mutually exclusive
+#     update_every: 3         # [optional] the JOB's data collection frequency
+#     priority: 60000         # [optional] the JOB's order on the dashboard
+#     retries: 60             # [optional] the JOB's number of restoration attempts
+#     timeout: 1              # [optional] the timeout when connecting, supports decimals (e.g. 0.5s)
+#     url: 'http[s]://host-ip-or-dns[:port][path]'
+#                             # [required] the remote host url to connect to. If [:port] is missing, it defaults to 80
+#                             #   for HTTP and 443 for HTTPS. [path] is optional too, defaults to /
+#     redirect: yes           # [optional] If the remote host returns 3xx status codes, the redirection url will be
+#                             #   followed (default).
+#     status_accepted:        # [optional] By default, 200 is accepted. Anything else will result in 'bad status' in the
+#                             #   status chart, however: The response time will still be > 0, since the
+#                             #   host responded with something.
+#                             #   If redirect is enabled, the accepted status will be checked against the redirected page.
+#       - 200                 # Multiple status codes are possible. If you specify 'status_accepted', you would still
+#                             #   need to add '200'. E.g. 'status_accepted: [301]' will trigger an error in 'bad status'
+#                             #   if code is 200. Do specify numerical entries such as 200, not 'OK'.
+#     regex: None             # [optional] If the status code is accepted, the content of the response will be searched for this
+#                             #   regex (if defined). Be aware that you may need to escape the regex string. If redirect is enabled,
+#                             #   the regex will be matched to the redirected page, not the initial 3xx response.
+
+# This plugin is intended for simple cases. Currently, the accuracy of the response time is low and should be used as reference only.
+
```

Comment:
i suggest to add several example jobs to the end of the file (commented out of course).